README.md

```diff
@@ -280,6 +280,22 @@ To get in touch with the developers,
 or [create an issue](https://github.com/trixi-framework/Trixi.jl/issues/new).
 
 
+## Participating research groups
+Participating research groups in alphabetical order:
```

Comment:
"Currently" somewhere?

Reply:
I'd leave this as it is to have a little more flexibility.